API-reference.md

```diff
@@ -155,6 +155,23 @@ In the portal, you can search on message content and [display individual trackTr
 (Unlike `trackEvent`, you can't filter on the message content in the portal.)
 
 
+### trackDependency
+
+    trackDependency(id: string, method: string, absoluteUrl: string, pathName: string, totalTime: number, success: boolean, resultCode: number) {
+    
+Log an ajax dependency
```

Comment:
Should we consider changing this from "Log an ajax dependency" to just "dependency" then.